src/Resources/config/user_context.xml

```diff
@@ -33,7 +33,7 @@
 
         <service id="fos_http_cache.user_context.logout_handler" class="FOS\HttpCacheBundle\Security\Http\Logout\ContextInvalidationLogoutHandler" public="false">
             <argument type="service" id="fos_http_cache.user_context_invalidator" />
-            <deprecated>The "%service_id%" service is deprecated since 2.2 and will be removed in 3.0.</deprecated>
+            <deprecated version="2.2" package="friendsofsymfony/http-cache-bundle">The "%service_id%" service is deprecated since 2.2 and will be removed in 3.0.</deprecated>
```

Comment:
sometimes supporting multiple symfony versions is really hard :-(

these attributes are required with symfony 6 but forbidden in older symfony versions. should we split the configuration file? does it even make sense to keep providing the legacy logout handler in symfony 6 or does it not work anyways?

Reply:
i split the file